### DISCUSSION TITLE
Question about commentary while validating annotation

### DISCUSSION THREAD

**Discussion by PotOfJam** [inception-project/inception#4820] [Q&A] [answered]
Hi!

I'm working on a document classification annotation campaign, and my users are complaining about the commentary pop-up that appears each time they want to validate a document (because 99.9% of the time they have nothing to comment).

Is there a way to disable it, or is there a key binding to quickly dismiss the pop-up when it appears?

thanks in advance

**Comment by reckart** [comment]
The primary function of the dialog is to warn the user that when they mark the document as finished, they cannot come back and work on it again without help from the manager. The comment function is just a little add-on.

You can avoid the dialog by giving the annotators the permission to re-open documents in **Project Settings -> Workload -> Allow annotators to re-open a document**.

**Reply by PotOfJam** [reply]
So it is not possible to disable it in dynamic mode ?

**Reply by reckart** [reply, marked as answer]
In dynamic mode, users can never go back to documents. Thus there is always the warning dialog, yes - there is currently no option to disable it. You can open a feature request though if you believe it would be a useful option for the dynamic mode as well.

Note, if you allow users to re-open documents, you should instruct your curators to not start curating until the all documents are fully annotated. If an annotator re-opens a document that a curator is already working on, it will be a moving target for the curator - not good.

**Reply by PotOfJam** [reply]
Okay thanks you ^^, i will open a features request
